test/Intrinsics/Saturating.ll

```diff
@@ -0,0 +1,80 @@
+; REQUIRES: geq-llvm-8.0
+; RUN: llvm-as %s -f -o %t1.bc
+; RUN: rm -rf %t.klee-out
+; RUN: %klee --output-dir=%t.klee-out --optimize=false %t1.bc > %t2
+; RUN: grep PASS %t2
```

Comment:
I can't find any similar tests that use FileCheck like this, would I just replace the grep line with the following?:
```
; RUN: FileCheck -check-prefix=PASS -input-file=%t2 %s
```

Reply:
You would do the following:
```
RUN: %klee --output-dir=%t.klee-out --optimize=false %t1.bc | FileCheck %s
```
instead of the last two RUN-lines and further down, ideally just above the corresponding `@puts` call, you write:
```
; CHECK: PASS
```
You can find the documentation to FileCheck here: https://llvm.org/docs/CommandGuide/FileCheck.html